src/import.js

```diff
@@ -96,9 +96,9 @@ export default function up() {
       await executor
         .withHttpPort(argv.port)
         .withBindAddr(argv.addr)
-        // only open  browser window when executable is `hlx`
+        // only open  browser window when executable is `aem`
         // this prevents the window to be opened during integration tests
-        .withOpen(path.basename(argv.$0) === 'hlx' ? argv.open : false)
+        .withOpen(path.basename(argv.$0) === 'hlx' || path.basename(argv.$0) === 'aem' ? argv.open : false)
```

Comment:
```suggestion
        .withOpen(path.basename(argv.$0) === 'aem' ? argv.open : false)
```
since it cannot be hlx anymore, first test is useless.